### DISCUSSION TITLE
Skipping a sector number on one track

### DISCUSSION THREAD

**Discussion by fbrown540** [keirf/flashfloppy#901] [General]
The TSC FLEX 2.0 disk format (Used in SWTPC 6800 computers) uses a special format on track 0.  Track 0-Head 0 is numbered 0,2,3,4,5,6,7,8,9,10.   Every other track  is numbered 1-10.  Is there any way to specify this in the IMG.CFG file.

**Comment by keirf** [comment]
At the moment, no, except that you could convert such a track using format `ibm.scan`. I will look into allowing to specify a list of ids for a track. I should allow override of all CHRN bytes in a sector header actually.
